configure.ac

```diff
@@ -632,7 +632,7 @@ echo
 echo '* Checking for apt_pkg library used by: dpkginfo '
 PKG_CHECK_MODULES([apt_pkg], [libapt-pkg >= 0.0],[],[
 SAVE_LIBS=$LIBS
-AC_SEARCH_LIBS([pkgInitConfig],[apt-pkg],[
+AC_SEARCH_LIBS([pkgVersion],[apt-pkg],[
```

Comment:
1) I would edit `ac_probes/configure.ac.tpl` similarly as you did with `configure.ac`.
2) Then I would run `./confgen.sh`. It will replace/regenerate configure.ac
3) `git add -p` and you are done.